Makefile

```diff
@@ -275,20 +278,25 @@ test-unit:
 #
 # Container tests and VM tests can be run in any order, but they must be run sequentially.
 # For more details, see the package-level comment of the e2e package.
+
+.PHONY: create_report_dir
+create_report_dir:
```

Comment:
Creating the target won't execute it. I would suggest adding this to the `test-e2e-vm*` targets below